docs: Update and describe ENV variables (#297)
epiphany-platform/CommonDataLayer: docs/architecture/data_router.md

> To configure data router, set following environment variables. `INPUT_ADDR` and `INPUT_QUEUE` is related to the incoming data in the router. `KAFKA_BROKERS`, `KAFKA_ERROR_CHANNEL` are related to messages being routed through Kafka to the corresponding command service.

Reviewer: ```suggestion
To configure the Data Router, set the following environment variables:
```

| Name | Short Description | Example | Mandatory | Default |
|---|---|---|---|---|

> | COMMUNICATION_METHOD | The method to communicate with external services | `kafka` / `amqp` / `grpc` | yes | |
> | INPUT_SOURCE | kafka topic or amqp queue | `cdl.data.input` | no, when `grpc` has been chosen | |
> | SCHEMA_REGISTRY_ADDR | Address of setup schema registry | http://schema_registry:50101 | yes | |
> | CACHE_CAPACITY | How many entries the cache can hold | 1024 | yes | |
> | TASK_LIMIT | limits the number of tasks | 128 | yes | 128 |
> | METRICS_PORT | Port to listen on prometheus requests | 58105 | no | 58105 |
> | RUST_LOG | Log level | `trace` | no | |

Reviewer: ```suggestion
| COMMUNICATION_METHOD | The method to communicate with external services | `kafka` / `amqp` / `grpc` | yes | |
| INPUT_SOURCE | Kafka topic or AMQP queue | `cdl.data.input` | no, when `grpc` has been chosen | |
| SCHEMA_REGISTRY_ADDR | Address of schema registry gRPC API | http://schema_registry:50101 | yes | |
| CACHE_CAPACITY | How many entries the cache can hold | 1024 | yes | |
| TASK_LIMIT | Max requests handled in parallel | 128 | yes | 128 |
| METRICS_PORT | Port to send metrics data to Prometheus over | 58105 | no | 58105 |
| RUST_LOG | Log level | `trace` | no | |
```

#### Kafka configuration 
(if `COMMUNICATION_METHOD` equals `kafka`)

| Name | Short Description | Example | Mandatory | Default |
|---|---|---|---|---|

> | KAFKA_BROKERS | Address to kafka brokers | `kafka:9093` | yes | |
> | KAFKA_GROUP_ID | Group id of the consumer | `data_router` | yes | |

Reviewer: ```suggestion
| KAFKA_BROKERS | Address of Kafka brokers | `kafka:9093` | yes | |
| KAFKA_GROUP_ID | Group ID of the consumer | `data_router` | yes | |
```

#### Amqp configuration 
(if `COMMUNICATION_METHOD` equals `amqp`)

| Name | Short Description | Example | Mandatory | Default |
|---|---|---|---|---|

> | AMQP_CONNECTION_STRING | Connection string to AMQP Server | `amqp://user:CHANGEME@rabbitmq:5672/%2f` | yes | |

Reviewer: ```suggestion
| AMQP_CONNECTION_STRING | Connection URL to AMQP Server | `amqp://user:CHANGEME@rabbitmq:5672/%2f` | yes | |
```

| AMQP_CONSUMER_TAG | Consumer tag | `data_router` | yes | |

#### GRPC configuration 
(if `COMMUNICATION_METHOD` equals `grpc`)

| Name | Short Description | Example | Mandatory | Default |
|---|---|---|---|---|

> |

Reviewer: Extra line here.
